nexus_constructor/geometry/off_geometry.py

```diff
@@ -38,7 +39,7 @@ def winding_order_indices(self) -> List[int]:
 
     @property
     @abstractmethod
-    def off_geometry(self) -> "OFFGeometry":
+    def mesh(self) -> OffMesh:
```

Comment:
I think the return type should be `Union(Qt3DExtras.QCylinderMesh, OFFMesh)`

Reply:
Or if `OFFMesh` and `QCylinderMesh` inherit from a base mesh class then that would be appropriate